onmt/io/AudioDataset.py

```diff
@@ -2,10 +2,12 @@
 
 import codecs
 import os
+import io
+from tqdm import tqdm
 
 import torch
 import torchtext
-
+import sys
 from onmt.io.DatasetBase import ONMTDatasetBase, PAD_WORD, BOS_WORD, EOS_WORD
```

Comment:
I would follow the practice of import order:  system, then third-party, and finally inner-project.